test/expected/timestamp.out

```diff
@@ -595,52 +595,3 @@ FROM unnest(ARRAY[
  11-09-2017 | 11-09-2017
 (4 rows)
 
-------------------------------------
```

Comment:
We should still be testing these conversions somewhere

Reply:
Yes but the associated conversion still does and should be tested.

Reply:
Are you suggesting we expose these functions publicly for the sake of testing? I don't mind testing but why single these functions out compared to a lot of other functions that we don't expose?

Reply:
We don't necessarily need to expose it but at least test all the edge cases by using set_interval or whatnot. These time conversion functions have had issues multiple times before so I think its a good idea to be preemptive and test these code paths.

Reply:
Ok, managed to add these back by exposing the function. Also figured out a way to only add the SQL part in the tests

Reply:
I really like this!